samples/CGCatalog/CGCatalog/Samples/CGCCGContextBeginEndTransparencyLayer.m

```diff
@@ -48,7 +47,7 @@ - (void)drawRect:(CGRect)rect {
     // Set composite alpha of the transparency layer
     // CGContextSetAlpha(context, 0.5);
```

Comment:
same here regarding referencing the GitHub issue.  We should probably also delete the commented out code for now.  Can you please add it to the GitHub issue, so we know to add the code back to these tests when the issue is fixed?